### PR TITLE
Added danish JSON translation

### PR DESCRIPTION
**Pull request by jensstigaard**

### CODE REVIEW

File: json/da.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "A fresh verification link has been sent to your email address.": "Et nyt verifikations-link er sendt til din email-adresse.",
+  "All rights reserved.": "All rights reserved.",
```
[the remark this answers is not in the document]
Comment (jensstigaard) [reply json/da.json]: Woops - forgot that one. I have just made a new commit to the repo for the pull request. It should be complete now, and ready to merge.

File: README.md [hunk 2]
```diff
@@ -10,15 +10,18 @@ In this repository, you can find the lang files for the framework PHP, [Laravel
 ## Install
 
 #### Via Composer
+Laravel version of your app
```
[the remark this answers is not in the document]
Comment (jensstigaard) [reply README.md:13]: Will do

File: json/da.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Hvis du har problemer med at klikke på \":actionText\", kan du kopiere nedenstående URL ind en web-browser:",
+  "Invalid signature.": "Ugyldig signatur.",
+  "Login": "Log ind",
+  "Logout": "Log ud",
+  "Name": "Navn",
+  "Not Found": "Ikke fundet",
+  "Oh no": "Oh no ",
+  "Page Expired": "Side udløbet",
+  "Page Not Found": "Siden blev ikke fundet",
+  "Password": "Password",
```
[the remark this answers is not in the document]
Comment (jensstigaard) [reply json/da.json:24]: Well in danish most people use the word password instead of adgangskode. Therefore I would stick to password.

**Comment by jensstigaard** [comment]
Once more I have updated the pull request with changes.

File: json/da.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (caouecs) [reply json/da.json:24]: Ok, I just compare with password.php, but ok ;)

**Comment by caouecs** [comment]
Please, delete your updates on README and I will merge

**Comment by jensstigaard** [comment]
Reverted readme changes. Ready for merge. @caouecs

**Comment by caouecs** [comment]
Tak

**Pull request merged**